examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<p>grammar: "an estimated ground state of -14"</p><p> </p><p>What about the envelop for GD and QNG? Also, what does the envelop mean?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Envelop refers to the overall profile of all 20-25 curves. Here the curves are started with the same initial config. GD and QNG are analytical optimizers. Up to the measurement uncertainty due to the finite shot, GD and QNG curves are supposed to be reproducible for each trial.